src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -123,27 +124,18 @@ private void initUI() {
         VBox milestoneDialogBox = createMilestoneDialogBox();
         assignedMilestoneBox = createAssignedMilestoneBox();
         matchingMilestonesBox = createMatchingMilestonesBox();
-        ScrollPane matchingMilestonesScrollPane = coverWithScrollPane(matchingMilestonesBox);
         inputField = createInputField();
 
-        milestoneDialogBox.getChildren().addAll(assignedMilestoneBox, inputField, matchingMilestonesScrollPane);
+        milestoneDialogBox.getChildren().addAll(assignedMilestoneBox, inputField, matchingMilestonesBox);
```

Comment:
Yes. This is in consideration of #1413, where a scrollable pane filled with all the milestones will be added. It will seem redundant/weird if the matching milestones pane is also scrollable. Or should we leave it as scrollable until #1413 is done?